client_app/src/components/shelter/ShiftDetailsData.tsx

```diff
@@ -97,19 +89,19 @@ export const availableShifts = {
     {
       id: 1,
       name: "Morning",
-      startTime: "10:00 AM",
-      endTime: "12:00 PM"
+      startTime: 1728572400000,
+      endTime: 1728586800000,
     },
     {
       id: 2,
-      startTime: "12:00 PM",
-      endTime: "6:00 PM"
+      startTime: 1728586800000,
+      endTime: 1728601200000
     },
     {
```

Comment:
In this shifts, maybe have an "Afternoon:" heading like you do for Morning and Evening for better reading readability

Reply:
The reason I have it rendered like this is because I was showing a use case when a shelter doesn't enter a name (because that requires different logic to be considered for rendering).